app/models/order_form.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class OrderForm
+  include ActiveModel::Model
+
+  attr_accessor :reward, :delivery_method, :employee, :street, :postcode, :city, :address_id
+
+  def save
+    r = Reward.find(reward)
+    ActiveRecord::Base.transaction do
+      if r.delivery_method == 'post'
+        if address_id.present?
```

Comment:
maybe move this to some helper method `address`?

Reply:
Good point!